### ISSUE TITLE
Unable to write command 2237452 to file (DeviceIoControl)

### ISSUE THREAD

**Issue by mstephenson6** [cronosun/atrofac#15]
Laptop: ROG GA502D (according to panel), GU502DU (according to OpenHardwareMonitor)
OS: Win10 202H2

Downloaded atrofac-gui.exe, tried running from Explorer, PowerShell, and as Admin. Keep getting this message on launch:

![image](https://user-images.githubusercontent.com/1770330/109810645-c9571c80-7bf7-11eb-944a-ef586e0ef458.png)

Not much in `C:\Users\<USERNAME>\AppData\Local\atrofac_logs`:

```
[2021-03-03 08:09:21.196207 -05:00] INFO [atrofac_libgui::engine::engine] libgui\src\engine\engine.rs:203: atrofac started.
[2021-03-03 08:09:21.196207 -05:00] INFO [atrofac_libgui::engine::engine] libgui\src\engine\engine.rs:60: Configuration file successfully (re-)loaded.
```

I have the popular "one rattling fan at low RPM" issue and I'm trying to just go fanless on battery.

Is there anything I can do to provide more info or get atrofac running?

**Comment by jsa2** [comment]
Hi, I had similar error, and found the culprit to be the missing ATKACPI driver which might get uninstalled if you remove other Asus applications and services?

**Comment by bdelpey** [comment]
Hi,
I have similar error and log files with an Asus TUF dash F15.
Tried to install ATKACPI driver but this didn't help.

I'm stuck with the 3 profiles mode of armory crate but I'd like to fine-tune the fans speed.
Was hoping this tool could help me.

Also tried without success to find older version of armory crate which had a manual option at the right of the turbo one according to screenshot I found.

Can someone help me make this tool work ?
